atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleTableInitializer.java

```diff
@@ -55,7 +55,7 @@ public void createUtilityTables() {
                 String.format(
                         "CREATE TABLE %s ("
                                 + "table_name varchar(2000) NOT NULL,"
-                                + "short_table_name varchar(30) NOT NULL,"
+                                + "short_table_name varchar(27) NOT NULL,"
```

Comment:
It can be derived from `OracleTableNameMapper.ORACLE_MAX_TABLE_NAME_LENGTH` and `AtlasDbConstants.PRIMARY_KEY_CONSTRAINT_PREFIX`.

Could be put in, but we might want to bind ourselves with a consistency test (assert that these two numbers combine to make 27), so that we don't change them and accidentally do something weird with schemas. Hmm, but probably a bunch of tests will fail anyway, if you change this number?